src/workflows-app/pages/SubmissionHistory.test.js

```diff
@@ -0,0 +1,672 @@
+import '@testing-library/jest-dom';
```

Comment:
We can get rid of this line - https://github.com/DataBiosphere/terra-ui/pull/4074#discussion_r1273758946